user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md

```diff
@@ -77,6 +77,17 @@ To configure TLS encryption for client-server communication:
    xpack.security.http.ssl.keystore.path: path/to/your/certificate
    xpack.security.http.ssl.truststore.path: path/to/your/certificate
    ```
+   Add the following lines to *elasticsearch.yml* file, if you are using `.PEM` certificate generated using `penssl utility` .
+   
+   ```
+    xpack.security.http.ssl.enabled: true
+    xpack.security.http.ssl.verification_mode: full
+    xpack.security.http.ssl.key: path/to/your/PrivateKey/used/to/generatet/the/certificate
+    xpack.security.http.ssl.certificate: path/to/your/certificate
+    xpack.security.http.ssl.certificate_authorities: path/to/certificate/authority (if you are using self-signed
```

Comment:
maybe add the last part as a comment (prefixed by '#') so the yaml stays valid if someone just copy pastes this